addons/project/data/mail_template_data.xml

```diff
@@ -97,5 +97,18 @@
             <field name="lang">{{ object.rating_get_partner_id().lang }}</field>
             <field name="auto_delete" eval="True"/>
         </record>
+
+        <!-- You have been assigned email -->
+        <template id="project_message_user_assigned">
+<p style="margin: 0px;">
+    <span>Dear <t t-esc="assignee_name"/>,</span><br />
+    <span style="margin-top: 8px;">You have been assigned to the <t t-esc="model_description or 'document'"/> <t t-esc="object.display_name"/>.</span>
+</p>
+<p style="padding-top: 24px; padding-bottom: 16px;">
+    <a t-att-href="access_link" t-att-data-oe-model="object._name" t-att-data-oe-id="object.id" style="background-color:#875A7B; padding: 10px; text-decoration: none; color: #fff; border-radius: 5px;">
+            View <t t-esc="model_description or 'document'"/>
+    </a>
+</p>
+        </template>
```

Comment:
Note for SM people (👋  TDE) :
I added a copy of the default template in project for two reasons:

1) It would try to output `object.user_id.sudo().name` which would result in an error

2) Using the regular system would not allow to have only one assignee per mail and it does not make any sense to send a mail to all assignees every time a new assignee changes

Reply:
Ok :) I don't have much time to look at it (freeze is coming). Anyway we can keep it localized in project currently, and see if we can / want to make something more generic afterwards in mail.

Reply:
Thanks for your quick review and bisoutas!